community/graphdb-api/src/main/java/org/neo4j/graphdb/security/WriteOperationsNotAllowedException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.graphdb.security;
+
+import org.neo4j.kernel.api.exceptions.Status;
+
+public class WriteOperationsNotAllowedException extends RuntimeException implements Status.HasStatus
+{
+    private Status statusCode = Status.General.DatabaseUnavailable;
```

Comment:
@pontusmelke How about this? https://github.com/neo4j/neo4j/pull/7948/commits/2623e3a26ba357325077f09b510b7beaecfe170f#diff-6bfb36f00f6d3f9f704f1617f43af598R485